playwright.config.ts

```diff
@@ -0,0 +1,108 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
Will you use a config file in the future, is that why this is here?

> Get the URL from an environment variable?

Anything else we would want to pop in there?

Reply:
> is that why this is here?

wellllll it _was put there_ by playwright when I initialized it....I _left it there_ because it seemed useful, if we decide to use it.

Reply:
Sounds good. Just remember to comment it back in when you try to use it later 😅